html/browsers/history/the-location-interface/location-define-own-property.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>[[DefineOwnProperty]] on a same-origin Location object behaves like ordinary</title>
```

Comment:
Title seems off. I think it's more, "[[DefineOwnProperty]] fails for toString and valueOf on same-origin Location objects", or maybe "same-origin Location objects have non-configurable toString and valueOf properties"?

Reply:
Thank you for taking a look; fixed test titles along with the file names in df0d3f6.